app/views/people/show.mobile.haml

```diff
@@ -10,7 +10,23 @@
     %span.description
       = @person.diaspora_handle
     - if user_signed_in? && @person != current_user.person
-      = render 'aspect_memberships/aspect_membership_dropdown'
+      - if @presenter.full_hash[:block]
+        .pull-right
+          = link_to t("users.privacy_settings.stop_ignoring"),
+            block_path(@presenter.full_hash[:block][:id]),
+            class: "btn btn-danger",
+            method: :delete
+      - else
+        = render "aspect_memberships/aspect_membership_dropdown"
+    .clear
+  .mobile-icon-bar
+    - if user_signed_in? && @person != current_user.person && !@presenter.full_hash[:block]
```

Comment:
You should move this line one up, because it currently displays an empty icon-bar (with an double line) when you are already ignoring a user:

![image](https://user-images.githubusercontent.com/458548/50739848-eba2aa80-11e5-11e9-9944-5dee06f0caa6.png)